QUEST.tsv

```diff
@@ -45,251 +45,251 @@ QUEST_20150317_000044	Items to be deleted
+QUEST_20150317_000254	The Panto were once docile, but now they charge and attack on sight.
+QUEST_20150317_000255	There is a tree here that the Panto worship. {nl}If this is happening, that tree has almost certainly been corrupted by a dark power.
+QUEST_20150317_000256	Please protect the Romuvan barrier at the top of the valley. {nl}We will take care of the rest when our brothers arrive here.
+QUEST_20150317_000257	May the Goddess bless you.
+QUEST_20150317_000258	I want to get some action. {nl}I haven't seen a fight in a long while.
 QUEST_20150317_000259	We're still preparing. {nl}Please wait a little longer.
 QUEST_20150317_000260	Antares
-QUEST_20150317_000261	What? You still try and hinder my experiments? {nl}I will not allow that.
-QUEST_20150317_000262	Be careful! I'm sure this is Antares' work. It'll explode if you get too close. {nl}It's better to get rid of them by drawing the monsters one by one. 
-QUEST_20150317_000263	When I release this valve, my grand experiment will be complete. {nl}You will witness a spectacular scene.
-QUEST_20150317_000264	Don't interrupt me while I stabilize the Jewel of Prominence.
-QUEST_20150317_000265	You still don't get it? If it weren't for Antares, I'd be spending my life as a human. {nl}That's right - I'm a Demon. 
-QUEST_20150317_000266	Indeed… the Jewel of Prominence was hard to make, and it would be nice to see the final product for once. {nl}It's naturally my favorite, but I don't know about you, hee-hee.
-QUEST_20150317_000267	Do you know about the Demon Queen, Helgacercle? {nl}I've read that the Mage Tower was attacked several times in the past. 
-QUEST_20150317_000268	She'll continue growing stronger until her envy wanes. {nl}Every time she comes, though, I wonder if it will be this time.
-QUEST_20150317_000269	Look out behind you!
-QUEST_20150317_000270	I hope this message will reach you under the blessings of fire. {nl}Here are the cataclysms I expect to happen in your age.
+QUEST_20150317_000261	What? You still try and disturb my experiments? {nl}I will not allow that!
+QUEST_20150317_000262	Be careful! I'm sure this is Antares' work. It'll explode if you get too close. {nl}You'd better lure the monsters and remove them one at a time.
+QUEST_20150317_000263	When I release this valve, my grand experiment will be complete and you'll see a most spectacular sight!
+QUEST_20150317_000264	Don't interrupt me while I stabilize the Jewels of Prominence.
+QUEST_20150317_000265	You still don't get it? If it weren't for Antares, I'd be spending my life as a human. {nl}That's right. I'm a demon.
+QUEST_20150317_000266	Indeed… the Jewel of Prominence was hard to make, and it would be nice to see the final product for once. {nl}It's naturally my favorite, but I don't know about you. Hahaha…
+QUEST_20150317_000267	Do you know about the Demon Queen, Helgasercle? {nl}I've read that she attacked the Magicians' Tower several times in the past. 
```

Comment:
https://github.com/ttgmichael/EnglishTranslation/wiki/2.2.%20Map

*(The dedicated contributors use this Wiki as an updated resource)*

Magicians' Tower -> Mage Tower